tests/e2e/browser/song/song.test.ts

```diff
@@ -0,0 +1,70 @@
+import { test, expect, Page } from "@playwright/test";
+
+import { gotoHome, navigateToMain } from "../../navigators";
+
+test.beforeEach(gotoHome);
+
+async function navigateToSong(page: Page) {
+  await navigateToMain(page);
+  await expect(page.getByText("ソング")).toBeVisible();
+  await page.getByText("ソング").click();
+}
+
+test("再生ボタンを押して再生できる", async ({ page }) => {
+  await navigateToSong(page);
+  // TODO: ページ内のオーディオを検出するテストを追加する
+
+  const getCurrentPlayhead = async () =>
+    await page.locator(".sequencer-playhead").boundingBox();
+
+  // 再生ボタンを押して再生
+  await page.locator(".sequencer-body").click({ position: { x: 107, y: 171 } });
```

Comment:
パッと見で意図がわからないので、コメント書いてもいいかもです...!
```suggestion
  await page.locator(".sequencer-body").click({ position: { x: 107, y: 171 } }); // ノートを追加
```

Reply:
確かにあった方がわかりやすいですね